underpants.go

```diff
@@ -174,10 +197,6 @@ type disp struct {
 
   // The OAuth configuration object needed for authentication.
   oauth *oauth.Config
-
-  // An LRU cache that maps raw cookie strings to full user object. This avoids having
-  // to authenticate and decode user cookies on each request.
-  cache *lilcache.Cache
```

Comment:
TBH, I haven't attempted a timing attack on it, but @JordanMilne pointed me to https://golang.org/src/runtime/hashmap_fast.go#L208 which does look somewhat suspicious.